src/Engine/Managers/CameraManager/CameraStorage.hpp

```diff
@@ -0,0 +1,54 @@
+#ifndef RADIUMENGINE_CAMERASTORAGE_HPP
+#define RADIUMENGINE_CAMERASTORAGE_HPP
+
+#include <Engine/RaEngine.hpp>
+#include <Engine/Renderer/Camera/Camera.hpp>
+
+namespace Ra {
+namespace Engine {
+class RenderParameters;
+} // namespace Engine
+} // namespace Ra
+
+namespace Ra {
+namespace Engine {
+
+/**
+ * Interface providing functions to access and store cameras in
+ * an implementation-defined way.
+ */
+class RA_ENGINE_API CameraStorage {
+    // TODO (Mathias) make camera storage compatible with range for ...
+  public:
+    CameraStorage() {}
+
+    /// Destructor
+    virtual ~CameraStorage() {}
+
+    /**
+     * Upload data to the GPU.
+     */
```

Comment:
Are you sure this has an interest ? What is uploaded to the GPU ? For doing what ?
The interface of lightmanager from which this were copied define this as, when rendering with many lights, we can upload all the lightdata on the GPU according to the Light shader we will use. Is there something similar here ?